kernel-version.mk

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2023 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# this file is included by Makefile
+# two variable are used as input for kernel build:
+# ZARCH - architecture (amd64, arm64)
+# PLATFORM - platform (generic, nvidia, rt, imx*)
+# based on these two variables we set kernel version, commit hash and branch
+# when new kernel is build and pushed to dockerhub corresponding
+# KERNEL_COMMIT_xxx variable must be manually updated
+
+KERNEL_COMPILER=gcc
+PLATFORM?=generic
+
+PLATFORMS_amd64=generic rt
+PLATFORMS_arm64=generic nvidia imx8mp_pollux imx8mp_epc_r3720 imx8mq_evk
+ARCHS=amd64 arm64
+
+# check that ZARCH is supported
+ifeq (, $(findstring $(ZARCH), $(ARCHS)))
+    $(error "Unsupported architecture $(ZARCH)")
+endif
+
+# check that PLATFORM is supported
+ifeq (, $(findstring $(PLATFORM), $(PLATFORMS_$(ZARCH))))
+    $(error "Unsupported combination of ZARCH=$(ZARCH) and PLATFORM=$(PLATFORM)")
+endif
+
+ifeq ($(ZARCH), amd64)
+    # either generic or rt
+    KERNEL_FLAVOR=$(PLATFORM)
+    KERNEL_VERSION=v6.1.38
+    KERNEL_COMMIT_generic=9f4ce3ddb
+    KERNEL_COMMIT_rt=deadbeef
```

Comment:
Commenting here just to remember you to update this hash....

Reply:
fixed and pushed